github/oidc.go

```diff
@@ -59,6 +53,12 @@ type OIDCToken struct {
 
 	// ActorID is the unique ID of the actor who triggered the build.
 	ActorID string `json:"actor_id"`
+
+	// Expiry is the expiration date of the token.
+	Expiry time.Time
```

Comment:
This was reported by `govet` and there was only a single class of issues reported: `fieldalignment: struct with 32 pointer bytes could be 24`.

I was also not super familiar with the error and did some reading: This check recommends having all pointers first in the struct, so that the garbage collector does not need to scan the whole struct to test all pointers.

It was a bit fiddley to figure out the correct order, since a struct seems to be implemented as pointer+int. Not sure if this is dependent on architecture/implementation or actually defined in the go language spec?

Not sure if some additional guidance in the documentation would be helpful if folks get tripped up by these types of errors? I'd be open to add some lines to the docs.

Reply:
Yeah, it seems to basically be space efficiency though maybe some benefits with regard to GC. I'm not sure that we're really worried about it and it may be more trouble than it's worth, but we can enable it and see how it goes.